revisions-information.md

```diff
@@ -0,0 +1,87 @@
+# Hackage Metadata Revisions — What They Are, How They Work
+
+## What are revisions?
+
+Package maintainers (as well as Hackage trustees) may provide metadata-revisions to existing package versions. These revisions can be thought of as "updating" the cabal file of a package with new or more current information. However, they do not _actually_ update the file. Tarballs retrieved from hackage are always as the authors uploaded them. Revision information is tracked explicitly in the Hackage index, and `cabal-install` will chose to prefer the latest revision of a cabal file to the original one when downloading or installing a package.
```

Comment:
Tarballs retrieved from *hackage* (should be Hackage, capitalised, as elsewhere).